plugins/backend/catalog-backend-module-okta/package.json

```diff
@@ -26,7 +26,7 @@
   },
   "backstage": {
     "role": "backend-plugin-module",
-    "pluginId": "catalog-backend-module-okta",
+    "pluginId": "catalog",
```

Comment:
I believe this would be a breaking change right and would warrant a `major` version changeset?